### DISCUSSION TITLE
Cname setting and not Propagating ?

### DISCUSSION THREAD

**Discussion by FreeSpirit63** [poweradmin/poweradmin#516] [General]
Seems I have issues getting the cnames to propagate and don't see why

I will add a screenshot of the settings

![Screenshot 07-06-2023 14 51 51](https://github.com/poweradmin/poweradmin/assets/10807760/ff1e1ab4-7f11-4b92-b0fe-44d17c727820)

**Comment by FreeSpirit63** [comment]
Also DKIM , SPF, and DMARC are not being seen shows no records found...

**Reply by FreeSpirit63** [reply]
https://www.whatsmydns.net/dns-lookup?query=rshosting.ltd&server=quad9

**Comment by FreeSpirit63** [comment]
Well the strange thing is that the record is created but only found as subdomain
![Screenshot 07-07-2023 07 01 11](https://github.com/poweradmin/poweradmin/assets/10807760/67f47af8-31d6-42cb-bfc5-41638dbba133)

**Comment by FreeSpirit63** [comment]
The _dmarc.rshosting.ltd is only found as subdomain and not in the Zone
![Screenshot 07-07-2023 07 05 47](https://github.com/poweradmin/poweradmin/assets/10807760/f7cf9289-22a6-44e9-9150-4bc839334cfd)

**Comment by FreeSpirit63** [comment]
PS: webmail, ftp, cpanel all work

**Comment by edmondas** [comment]
Closing this old one. The screenshots show the records did resolve: Quad9 returned the mail CNAME and Cloudflare returned the _dmarc TXT record. Querying the apex for TXT won't return `_dmarc`, it lives on its own name.

Two things in the zone itself were wrong though, in case someone finds this later:

- The SPF policy was stored as an `SPF` record type. That type is deprecated (RFC 7208) and mail servers only check TXT, so put it in a TXT record instead.
- The DKIM record was named `dkim_domainkey` instead of `dkim._domainkey`, so DKIM lookups could never find it.
